### PR TITLE
fixed bug with array of undefined values

### PR DESCRIPTION
**Pull request by midknight41**
Addresses #53

### CODE REVIEW

**Comment by midknight41** [comment]
@simranjeet2589 Can you have a look at this?

File: src/lib/mapper.js [hunk 1]
```diff
@@ -267,6 +267,21 @@ export default class Mapper {
       Object.keys(object).length === 0;
   }
 
+  handleArrayOfUndefined_(value) {
```
Comment (simranjeet2589) [inline src/lib/mapper.js]: Do we need this function in this file?

Comment (midknight41) [inline src/lib/mapper.js]: Removed.

Comment (midknight41) [inline src/lib/mapper.js]: Removed.

**Pull request merged**